kong/pdk/response.lua

```diff
@@ -1034,29 +1039,31 @@ local function new(self, major_version)
   -- return kong.response.error(500, "Error", {["Content-Type"] = "text/html"})
   -- ```
   --
-  -- The first argument `status` will set the status code of the response that
-  -- will be seen by the client. The status code must be of an error, i.e.
-  -- >399.
+  -- 1. The `status` argument sets the status code of the response that
+  -- is seen by the client. The status code must an error code, that is,
+  -- greater than 399.
   --
-  -- The second, optional, `message` argument will set the message describing
-  -- the error, which will be written in the body.
+  -- 1. The optional `message` argument sets the message describing
+  -- the error, which is written in the body.
   --
-  -- The third, optional, `headers` argument can be a table specifying response
+  -- 1. The optional `headers` argument can be a table specifying response
```

Comment:
Should these be `2.` and `3.`?

Reply:
They don't need to be, the generator sees a numbered list and numbers it accordingly. So it doesn't matter what numbers are used here. In the docs, we're trying to use only `1.` going forward, so that switching steps around, removing, or adding steps, doesn't change the list in the output - here's the list in question in the preview: https://deploy-preview-3546--kongdocs.netlify.app/gateway/2.7.x/pdk/kong.response/#kongresponseexitstatus-body-headers

However - in this case, since the code comments are seen by PDK users, I can see an argument for actually numbering them 1, 2, 3.

Reply:
Ah okay, gotcha. Yeah, I realize it's less convenient in the doc-editing context, but I do feel like we want to maintain readability in the source.